### ISSUE TITLE
Icons Broken in ML4W dotfiles settings

### ISSUE THREAD

**Issue by Neo-29** [mylinuxforwork/dotfiles-settings#4]
![image](https://github.com/user-attachments/assets/e2de47ce-1b92-4b63-9b31-f5a4365ccc9a)

See image

**Comment by mylinuxforwork** [comment]
@Neo-29 ON my minimal arch system everything is working. Can you open nwg-look and set a GTK theme?

**Comment by Neo-29** [comment]
> @Neo-29 ON my minimal arch system everything is working. Can you open nwg-look and set a GTK theme?

What do you mean?
I set an icon theme (adwaita)
I don't know what you mean or where to put a gtk theme in `nwg-look`

**Comment by mylinuxforwork** [comment]
@Neo-29 Yes. That's the reason. With all my icon themes it's working fine except adwaita. 
Just open rofi and search for nwg-look, The GTK App should appear. There you can set the icon theme.

![image](https://github.com/user-attachments/assets/65d8df7f-888d-40b2-9ca2-515689563cb9)

**Comment by Neo-29** [comment]
> @Neo-29 Yes. That's the reason. With all my icon themes it's working fine except adwaita. Just open rofi and search for nwg-look, The GTK App should appear. There you can set the icon theme.
> 
> ![image](https://private-user-images.githubusercontent.com/145253254/380709551-65d8df7f-888d-40b2-9ca2-515689563cb9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMTgyMzksIm5iZiI6MTczMDExNzkzOSwicGF0aCI6Ii8xNDUyNTMyNTQvMzgwNzA5NTUxLTY1ZDhkZjdmLTg4OGQtNDBiMi05Y2EyLTUxNTY4OTU2M2NiOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOFQxMjE4NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYTYzNGJmOGZmODAxMTc0MzY5ZmIxOGRiNjkyMTI1ODM2ZGM3ZmFlNjk3ZDI3NDllZTllZTE2ZWE2YmY2YTk5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.0P-LlE35IKNkDN9M1Npz6bB1QoJ_D9GaOTw1EWtZ3mY)

I think it will be not the only one breaking.
Breeze is one too.
I'll test others and let you know, but there's obviously something to "correct" somewhere.
Tks

**Comment by Neo-29** [comment]
![image](https://github.com/user-attachments/assets/3da9d0c8-70fa-4fbc-88bb-0c7d911b15aa)

whitesur here...no issue

Strange it's only `adwaita` and `breeze`
